### PR TITLE
fix(fast-data): add missing steps to configure sv-patch on the svc

### PR DESCRIPTION
**Pull request by ianFar96**
## Description

Add missing steps on the configuration of the sv-patch in the Single View Creator

## Pull Request Type

- [X] Documentation content changes
- [ ] Bugfix / Missing Redirects
- [ ] Docusaurus site code changes

## PR Checklist

- [X] The commit message follows our guidelines included in the [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md#how-to-submit-a-pr)
- [X] All tests of Lint, cspell and check-content pass. ([How to launch them?](../blob/main/CONTRIBUTING.md#content-checks))
- [X] No sensible content has been committed

### CODE REVIEW

File: docs/fast_data/configuration/single_view_creator.md [hunk 1]
```diff
@@ -403,8 +403,10 @@ This feature is supported from version `5.6.1` of the Single View Creator
 To configure a Single View Creator dedicated to [Single View Patch](/fast_data/configuration/single_views.md#single-view-patch) operations, some steps have to be followed:
 
 * Set the env var `KAFKA_PROJECTION_UPDATE_TOPICS` with the comma separated list of the `pr-update` topics corresponding to the SV-Patch Projection.
+* Set the env var `PROJECTIONS_CHANGES_SOURCE` to `KAFKA` so the consumer is summoned.
+* Configure the environment variables for kafka (see the [environment variables table](#environment-variables))
```
Comment (TheRusCompass) [inline docs/fast_data/configuration/single_view_creator.md:407]: which are all the KAFKA env vars? are they stated somewhere?

Comment (ianFar96) [reply docs/fast_data/configuration/single_view_creator.md]: You're right I added a paragraph explaining what is needed

Comment (TheRusCompass) [reply docs/fast_data/configuration/single_view_creator.md]: is it done?

File: docs/fast_data/configuration/single_view_creator.md [hunk 2]
```diff
@@ -150,6 +150,17 @@ service.addHook('onClose', async() => {
...
+
+- You want to use the [Single View Trigger Generator](/fast_data/single_view_trigger_generator.md) to produce [`sv-trigger`](/fast_data/inputs_and_outputs.md#single-view-trigger-message) messages.
+- You want to configure the [Single View Patch](#single-view-patch) cycle which reads [`pr-update`](/fast_data/inputs_and_outputs.md#projection-update-message) messsages from the Real Time Updater.
+
+In both of the cases you have to configure all the required environment variables related to kafka. First you need to configure the `KAFKA_BROKERS` and `KAFKA_GROUP_ID`, then you probably need to configure your authentication credentials with `KAFKA_SASL_MECHANISM`, `KAFKA_SASL_USERNAME` and `KAFKA_SASL_PASSWORD`.
+
+Once this is done remember to set the `PROJECTIONS_CHANGES_SOURCE` environment variable to `KAFKA` and to check out the configuration page of the system you need to complete the necessary steps.
```
Comment (TheRusCompass) [inline docs/fast_data/configuration/single_view_creator.md:162]: let's link the configuration page, if possible on the specific paragraph

Comment (ianFar96) [reply docs/fast_data/configuration/single_view_creator.md:162]: They are right above, since there are two cases I cannot put a link to an exact page

**Review by TheRusCompass — APPROVED** [review]

**Pull request merged**